src/components/home.vue

```diff
@@ -1,21 +1,362 @@
 <template>
-    <div>
-
+  <div class="container">
+    <h3>Most Popular</h3>
```

Comment:
Create a component called `MovieAndTvShowList` and inside that component move the logic for fetching and displaying the data. Pass arguments `movieEndpointType` and `tvShowEndpointType` for fetching certain type of movies/tvshows and `title` for displaying type of the list e.g. `Most Popular`. I am aware that types for movies and tvshows are not exactly the same but you can combine them like this:
1. Most Popular -- Most Popular
2. Now playing -- Airing today
3. Upcoming -- On TV
4. Top rated -- Top rated

Sorry for the confusion but I did not designed the exercise perfectly. Only Allah is perfect.